SPECS/aws-nitro-enclaves-cli.spec

```diff
@@ -195,6 +195,45 @@ fi
 %{ne_include_dir}/*
 
 %changelog
+* Tue Apr 16 2024 Erdem Meydanli <meydanli@amazon.com> - 1.3.0-0
++ This release focuses on resolving two critical issues:
+the vsock-proxy DNS lookup limitation (#553) and the compatibility
+problem with Docker versions 25 and later (#591). Furthermore, it
+updates several important crate dependencies to their latest versions.
+- cargo: Update cargo.lock to eliminate build failures
+- build(deps): bump base64 from 0.21.4 to 0.22.0
```

Comment:
May be just aggregated view of all the dependencies update in a single line instead for readability purposes?
I am in favor of more human readable description of what was changed =)